common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -906,47 +907,76 @@ div.problem {
 // +Problem - Actions
 // ====================
 div.problem .action {
-  margin-top: $baseline;
+  @include margin($baseline 0);
+  min-height: $baseline;
 
-  .save, .check, .show, .reset, .hint-button {
-    @include margin-right($baseline/2);
-    margin-bottom: ($baseline/2);
-    height: ($baseline*2);
-    vertical-align: middle;
-    text-transform: uppercase;
-    font-weight: 600;
-    @media print {
-      display: none;
+  // override default button hover
+  button {
+    &:hover {
+      background-image: none;
+      box-shadow: none;
+    }
+    &:focus {
+      box-shadow: none;
+    }
+    &.btn-default {
+      background-color: transparent;
+    }
+    &.btn-brand {
+      &:hover {
+        background-color: $btn-brand-focus-background;
+      }
+      &.check {
+        &.is-disabled {
+          background: $btn-brand-disabled-background;
+          color: $btn-brand-disabled-color;
+        }
+      }
     }
   }
 
-  .save {
-    @extend .blue-button !optional;
+  @media (min-width: 768px) {
+
+    .problem-action-buttons-wrapper {
+      position: absolute;
+      right: $baseline*1.5;
```

Comment:
We don't currently have a styleguide for sass, but the [sass docs use spaces around operators](http://sass-lang.com/guide#Operators), so IMO that's what we should be doing.